src/js/Framework/GoBack.js

```diff
@@ -0,0 +1,14 @@
+export class GoBack {
+  constructor () {
+    this.initEventListeners()
+  }
```

Comment:
Gingen we deze ook niet hernoemen? Weet het niet meer eigenlijk, had net het idee om het History te noemen, omdat hier dan alles dat met de history manager van u browser werkt hierin kan

Reply:
hmm zou dat niet doen. Idealiter pas ik dit es aan zodat het steeds naar een niveau hoger gaat, maar dat is voor een volgende release